pkg/kyma/cmd/provision/minikube.go

```diff
@@ -179,16 +184,12 @@ func (o *MinikubeOptions) Run() error {
 }
 
 func checkIfMinikubeIsInitialized(o *MinikubeOptions) error {
-	statusText, err := internal.RunMinikubeCmdE([]string{"status", "-b=" + bootstrapper, "--format", "'{{.MinikubeStatus}}'"})
```

Comment:
Shouldn't we keep the bootstrapper argument? yes, we are using the default, but then we should delete it from start command as well and delete the variable..

Reply:
I'll add it back